### ISSUE TITLE
TypeError occurs when answering yes/no question

### ISSUE THREAD

**Issue by frostming** [laixintao/iredis#311]
```pytb
iredis  1.4.3 (Python 3.6.8)
redis-server  3.2.8
Home:   https://iredis.io
Issues: https://iredis.io/issues
(IP) > keys *
KEYS will hang redis server, use SCAN instead.
Do you want to proceed? (y/n): n
a bytes-like object is required, not 'FormattedText'
Traceback (most recent call last):
  File "/root/.local/pipx/venvs/iredis/lib/python3.6/site-packages/iredis/entry.py", line 177, in repl
    write_result(answer)
  File "/root/.local/pipx/venvs/iredis/lib/python3.6/site-packages/iredis/entry.py", line 88, in write_result
    sys.stdout.buffer.write(text)
TypeError: a bytes-like object is required, not 'FormattedText'
(error) a bytes-like object is required, not 'FormattedText'
```

**Comment by laixintao** [comment]
Did you turn the `--raw` mode on? I can reproduce it under `--raw` mode, otherwise it works good.

<img width="689" alt="image" src="https://user-images.githubusercontent.com/9675939/77510404-a382ad00-6ea9-11ea-84f3-cb8efb8e6c44.png">

**Comment by frostming** [comment]
@laixintao Oh yes, I did use `--raw`

**Comment by laixintao** [comment]
It's a bug, in `--raw` everything is expected to output as bytes to stdout, but the warnings here are FormattedText which not handle raw mode correctly.

I will fix it ASAP. Thanks for reporting.

**Comment by laixintao** [comment]
Released in v1.6, please upgrade and see if it still has the bug :)

**Comment by frostming** [comment]
The original issue is gone, but another occurs:
```
iredis  1.6.0 (Python 3.6.8)
redis-server  3.2.8
Home:   https://iredis.io
Issues: https://iredis.io/issues
9.146.192.98:6381> keys *
KEYS will hang redis server, use SCAN instead.
Do you want to proceed? (y/n): n   <-- Look at here, I typed 'n'
key1
key2
key3
```

**Comment by laixintao** [comment]
Sorry for that, fixed in v1.6.1, please try.

May I ask why do you use `--raw` a lot? :)

**Comment by frostming** [comment]
Not intentional, just copied the command from others.
